src/org-pending/store/reducers/org-pending.reducer.ts

```diff
@@ -1,47 +1,57 @@
-import { PendingOrganisation } from '../../models/pending-organisation';
+import { PendingOrganisation, PendingOrganisationSummary } from '../../models/pending-organisation';
 import * as fromRoot from '../../../app/store/reducers/app.reducer';
 import { createFeatureSelector, createSelector } from '@ngrx/store';
 import { PendingOrgActions, PendingOrgActionTypes } from '../actions/org-pending.actions';
 
-export interface State extends fromRoot.AppState {
-    pendingdOrgs: PendingOrganisationState;
-}
-
 export interface PendingOrganisationState {
-    showPendingOrg: boolean;
-    currentOrg: PendingOrganisation;
     pendingOrganisations: PendingOrganisation[];
+    loaded: boolean;
+    loading: boolean;
 }
 
 const initialState: PendingOrganisationState = {
-    showPendingOrg: true,
-    currentOrg: null,
-    pendingOrganisations: []
+    pendingOrganisations: null,
+    loaded: false,
+    loading: false
 };
 
-export function reducer(state = initialState, action: PendingOrgActions): PendingOrganisationState {
+export function reducer(
+    state = initialState,
+    action: PendingOrgActions
+    ): PendingOrganisationState {
     switch (action.type) {
-
-        case PendingOrgActionTypes.TogglePendingOrgCode:
-            console.log('existing state: ' + JSON.stringify(state));
-            console.log('payload' + action.payload);
+        case PendingOrgActionTypes.LOAD_PENDING_ORGANISATIONS: {
+            console.log('in load');
             return {
-                ...state,
-                showPendingOrg: action.payload
+              ...state,
+              loaded: false,
+              loading: true
             };
-
-        case PendingOrgActionTypes.SetCurrentPendingOrg:
+          }
+        case PendingOrgActionTypes.LOAD_PENDING_ORGANISATIONS_SUCCESS:
+        console.log(' action.payload pending',  action.payload);
+        const payload = action.payload;
+        let pendingOrganisations = payload;
+        if (pendingOrganisations.length !== 0) {
+            pendingOrganisations = payload.map((entity: PendingOrganisation) => {
```

Comment:
i think something is not right here... grab me when u start looking at it.. maybe we can pair up